src/client/inputhandler.h

```diff
@@ -456,10 +384,8 @@ class RandomInputHandler : public InputHandler
 		mousepos += mousespeed;
 	}
 
-	s32 Rand(s32 min, s32 max)
-	{
-		return (myrand()%(max-min+1))+min;
-	}
+	s32 Rand(s32 min, s32 max);
```

Comment:
Was thinking about the overhead but it's apparently only used 9 times each step. Ok.